### ISSUE TITLE
Docker: should I persist checkpoint file?

### ISSUE THREAD

**Issue by OleksiL** [toluaina/pgsync#304]
Documentation says that `pgsync` uses `checkpoint` file in order to keep track of what has been already synced from db to Elastic.
I have a question related to it for the case of running it in Docker - should I put this file into volume to make it persist between container runs to avoid re-syncing entire database after container was shut down?

Thanks.

**Comment by toluaina** [comment]
- You should definitely persist the checkpoint file and I have considered making it not hidden/more visible for this reason.
- You can use a network mount location. 
- We could also consider moving this state information to redis?
- The file system seems like a good idea at the time

**Comment by superbiche** [comment]
Storing it on Redis could help when deploying to Kubernetes, as for now I'm using a PersistentVolume just for this one file.

**Comment by loren** [comment]
I agree that storing this in Redis versus a PersistentVolume file is preferable

**Comment by toluaina** [comment]
Yes Redis seems like a good idea. I will add this to the TODO

**Comment by Phaze1D** [comment]
Could I  use AWS S3 for this?
